dockerfiles/enclave.Dockerfile

```diff
@@ -1,31 +1,39 @@
-FROM ghcr.io/edgelesssys/ego-dev:latest
+FROM ghcr.io/edgelesssys/ego-dev:latest AS build-base
 # on the container:
 #   /home/obscuro/data       contains working files for the enclave
 #   /home/obscuro/go-obscuro contains the src
 #
```

Comment:
This should be updated. Like specified that it's the struct of the execution stage not to be confused with the build stage.